src/mongocxx/test/bulk_write.cpp

```diff
@@ -25,6 +25,8 @@
 using namespace mongocxx;
 
 TEST_CASE("a bulk_write will setup a mongoc bulk operation", "[bulk_write]") {
+    instance::current();
```

Comment:
I can't think of one. It was a pain to add, but hopefully it just becomes part of how you write a test.